packages/rum-core/src/performance-monitoring/capture-navigation.js

```diff
@@ -119,60 +120,57 @@ function createResourceTimingSpan(resourceTimingEntry) {
   return span
 }
 
-function createResourceTimingSpans(entries, filterUrls, trStart, trEnd) {
+/**
+ * Checks if the span is already captured via XHR/Fetch patch by
+ * comparing the given resource startTime(fetchStart) aganist the
+ * patch code execution time.
+ */
+function isCapturedByPatching(resourceStartTime, requestPatchTime) {
+  return requestPatchTime != null && resourceStartTime > requestPatchTime
+}
+
+/**
+ * Check if the given url matches APM Server's Intake
+ * API endpoint and ignore it from Spans
+ */
+function isIntakeAPIEndpoint(url) {
+  return /intake\/v\d+\/rum\/events/.test(url)
```

Comment:
Can you comment (on the PR) why `isIntakeAPIEndpoint` is necessary given that before bootstrap time there can not be any api request since we haven't bootstrapped?